dateparser/timezones.py

```diff
@@ -6,7 +6,7 @@
 timezone_info_list = [
     {
         'regex_patterns':
-            [r'(\W|\d|_)\(?%s\)?$'],
+            [r'(\W|\d|_)\(?%s(\)|$|[^\w+])'],
```

Comment:
If we don't match it separately, the resultant date string (stripped of timezone info) looks like below. 
```
In [3]: regex  = re.compile(r'(\W|\d|_)PDT($|\W)', re.IGNORECASE)

In [4]: regex.sub(r'\1', '08/17/14 17:00 (PDT)')
Out[4]: '08/17/14 17:00 ('
```

For which, dateutil parser returns `Unknown string format` error and hence no date is parsed. So, either we can exclude `(` from the match group or replace it from the string. Either at [dateparser](https://github.com/scrapinghub/dateparser/blob/master/dateparser/date_parser.py#L174) or [timezone parser](https://github.com/scrapinghub/dateparser/blob/master/dateparser/timezone_parser.py#L12).